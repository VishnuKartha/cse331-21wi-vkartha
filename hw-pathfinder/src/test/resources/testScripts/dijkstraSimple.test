# Find a path with Dijkstra's algorithm with a small graph

CreateGraph smallGraph

AddNode smallGraph A
AddNode smallGraph B



AddEdge smallGraph A B 2.0
AddEdge smallGraph B A 10000000.0





FindPath smallGraph A B
