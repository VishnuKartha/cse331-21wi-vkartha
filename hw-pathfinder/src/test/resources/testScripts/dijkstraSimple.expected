# Find a path with Dijkstra's algorithm with a small graph

created graph smallGraph

added node A to smallGraph
added node B to smallGraph



added edge 2.000 from A to B in smallGraph
added edge 10000000.000 from B to A in smallGraph





path from A to B:
A to B with weight 2.000
total cost: 2.000